src/software/ai/hl/stp/tactic/chip/chip_fsm_test.cpp

```diff
@@ -26,14 +26,13 @@ TEST(ChipFSMTest, test_transitions)
         boost::sml::state<GetBehindBallFSM::GetBehindBallState>));
 
     // Robot is now behind ball
-    robot = ::TestUtil::createRobotAtPos(Point(-2, 1.8));
+    robot = ::TestUtil::createRobotAtPos(Point(-2, 1.7));
```

Comment:
curious why this chagned?

Reply:
I reduced the size of the getbehindball triangle, since it wasn't kicking properly. This was due to rotation being slightly slower in simulation because the radius is slightly larger than before (0.085->0.09, i.e. use max radius instead of an out of date measure value)